### PR TITLE
Favorites

### PR DESCRIPTION
**Pull request by gustavlrsn**
This PR adds the ability to favorite/star camps.

### CODE REVIEW

File: app/controllers/camps_controller.rb [hunk 1]
```diff
@@ -105,6 +105,27 @@ def join
...
+  def toggle_favorite
+    if !current_user
+      flash[:notice] = "please log in :)"
+    elsif @camp.favorite_users.include?(current_user)
+      @camp.favorite_users.delete(current_user)
+      redirect_to camp_path(@camp)
+
+    else
+      puts('===========================================================================')
+      puts(@user)
```
Comment (aerugo) [inline app/controllers/camps_controller.rb]: Are these debugging prints?

File: app/views/users/me.html.erb [hunk 2]
```diff
@@ -3,11 +3,21 @@
   <%= javascript_include_tag 'blueimp_script' %>
 <% end %>
 
-<%= render 'camps/dont_miss_out', except: nil %>
-<div class="me col-md-5 col-xs-12" dir="<%=t :html_direction %>" style="float: <%=t :lang_direction%>">
+<%
+=begin%>
+ <%= render 'camps/dont_miss_out', except: nil %> 
+<%
+=end%>
```
Comment (aerugo) [inline app/views/users/me.html.erb]: Looks like there are some strange linebreaks in the code here, maybe a prettify-error?

File: app/controllers/camps_controller.rb (hunk 1 above)
Comment (gustavlrsn) [reply app/controllers/camps_controller.rb]: Good catch, fixed!

File: app/views/users/me.html.erb (hunk 2 above)
Comment (gustavlrsn) [reply app/views/users/me.html.erb]: Fixed!

**Pull request merged**